src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/MediaCapturedEventArgs.shared.cs

```diff
@@ -54,6 +54,12 @@ ImageSource GetImageSource()
 			return !string.IsNullOrEmpty(path) ? path : null;
 		}
 
-		XCT.FileMediaSource GetMediaSource() => !string.IsNullOrEmpty(path) ? path : null;
+		XCT.FileMediaSource? GetMediaSource()
+		{
+			if (path != null && !string.IsNullOrEmpty(path))
```

Comment:
we don't this null check here.

Reply:
I agree, however the compiler doesn't. 

When I remove `path != null`, the roslyn still thinks that `path` may be `null`.

It's a bug in Roslyn that we'll just have to work around for now.

Reply:
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/13558917/110188497-be0b1900-7dd0-11eb-8fcc-730784b0172e.png">
🤷‍♂️

Reply:
I know, I added that comment somewhere. but you can use the `!`, no?